pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -101,7 +104,20 @@ type Reconciler struct {
 	tracker           tracker.Interface
 	configStore       configStore
 	timeoutHandler    *reconciler.TimeoutSet
+<<<<<<< HEAD
+
+	// The clock for tracking time
+	clock clock.Clock
+
+	// PipelineRuns that the controller will check its TTL and attempt to delete when the TTL expires.
```

Comment:
`PipelineRuns whose TTL the controller will check and attempt to delete when it expires`

Reply:
Thx :）